benchmarks/Cargo.toml

```diff
@@ -18,14 +18,14 @@
 [package]
 name = "datafusion-benchmarks"
 description = "DataFusion Benchmarks"
-version = "5.0.0"
+version = "11.0.0"
```

Comment:
If we plan on keeping this up to date, we should update the script at `dev/update_datafusion_versions.py`

Reply:
done, and I updated datafusion-examples as well .. makes sense to keep all the child code on the same version.